google/cloud/bigquery/v2/minimal/internal/json_utils.h

```diff
@@ -49,6 +49,15 @@ void SafeGetTo(ResponseType& value, nlohmann::json const& j,
   if (i != j.end()) i->get_to(value);
 }
 
+template <typename C, typename T, typename R>
+void SafeValueOr(nlohmann::json const& j, std::string const& key,
```

Comment:
nit: I don't think the name reflects what the function does. (It does not set a **value or** a default).

I think you can just call it `SafeGetTo`.